### ISSUE TITLE
Linux Postman app show up with blacked-out window on Ubuntu 14.04

### ISSUE THREAD

**Issue by MaRuifeng** [a85/POSTMan-Chrome-Extension#2437]
1. Postman Version: Postman-linux-x64-4.8.0.tar.gz
2. App (Chrome app or Mac app): Linux App
3. OS details: ubuntu-14.04.4-desktop-amd64 (in Oracle VirtualBox)
4. Is the Interceptor on and enabled in the app: N.A.
5. Did you encounter this recently, or has this bug always been there: First time
6. Expected behaviour: N.A.
7. Console logs (http://blog.getpostman.com/2014/01/27/enabling-chrome-developer-tools-inside-postman/ for the Chrome App, View->Toggle Dev Tools for the Mac app): N.A.
8. Screenshots (if applicable)

Steps to reproduce the problem:
1. Download the Linux Postman app from https://www.getpostman.com/apps
2. De-package and double click the Postman icon
2. A window shows up but with internal contents blacked out

**Comment by numaanashraf** [comment]
@MaRuifeng Thanks for reporting this. Would it be possible to open the app's devtools via 'View > Toggle DevTools' and send us a screenshot of any errors there?

**Comment by MaRuifeng** [comment]
@numaanashraf I am unable to do so as the menu buttons are not responding (or they are responding, but the dropdown does not show). I am also trying to check if anything goes wrong with my Ubuntu OS, as the Chrome App version of Postman does not work either.

**Comment by MaRuifeng** [comment]
@numaanashraf The issue got resolved when I unchecked the 'Enable 3D Acceleration' option under the Display tab on my VirtualBox VM settings. A dumb issue but it took me quite a while to resolve, as the Chrome app used to work properly even with the option enabled. 

Sorry for the trouble but perhaps a lesson for VirtualBox users.

**Comment by ghost** [comment]
Disabling 3D Acceleration for the whole vbox cannot considered as a SOLUTION! 

Disabling 3D Acceleration lead to have 100% of CPU used on the host!  just for the Ubuntu rendering.

For Example Visual Studio Code has a flag in which you can start it without the GPU acceleration...  but just for this process.    Why do not use the same approach so that we can still use 3D acceleration for the rest of the programs which works with that feature!   in the meanwhile the issue is really fixed.

**Comment by mikob** [comment]
Can we reopen this issue? I'm having the same problem. Can't do anything, just a black rectangle running Linux Mint 17.1 on a Windows 7 host with 3d acceleration enabled.

**Comment by taufikp** [comment]
+1  ... Postman only shows black background. Running Ubuntu 16.04 in VirtualBox with 3D acceleration. Setting the 3D Acceleration off will raise the CPU usage.

**Comment by sdnts** [comment]
This looks like an [issue](https://github.com/electron/electron/issues/4380) with Electron. We're actively watching the thread and will let you know here when we make progress.

**Comment by taufikp** [comment]
Yeah I also agree that Electron is the problem. Atom for Linux also has exactly the same problem when 3D acceleration in VirtualBox is enabled.

**Comment by beevor** [comment]
Yes, I want to confirm on Ubuntu 16.04 in a VM that only a black background shows up, freezing the VM . Disabling 3D acceleration 'fixes' the problem.

**Comment by sdnts** [comment]
Looks related to #2979, closing as duplicate. Please let me know if you disagree, I will reopen!

**Comment by Amrutasri** [comment]
I am using Ubuntu 18.04 and don't have any Virtual box enabled. Still I am facing the same issue. It keeps happening window shows up but with a blank screen and I had to reinstall it again.

**Comment by gkirill** [comment]
I have started to having same problem just today on Ubuntu 18.04. It looks like it was updated yesterday on March 9, 2020, and after that I just get blank screen, though everything worked just fine before the update. I tried to roll back with snap to version 6.7.6, but that did not help.

**Comment by numaanashraf** [comment]
@gkirill Could you let us know your app version?

**Comment by gkirill** [comment]
@numaanashraf The first time when I noticed the issue I checked version and it was 7.20 (if I recall correctly), and the latest updates was done on March 9th (again as far as I remember). Then I downgraded to 6.7.6 via command line. But it did not solve the issue. Then I kindof gave up, but now I tried it again and it seems to work again. My version is still 6.7.6.

**Comment by pigubaoza** [comment]
I am having this problem on Ubuntu 16.04, and I'm not running it as a VM. It would start up fine, then black out after a few seconds without any user input. v7.22.1

**Comment by indrkl** [comment]
Same here, postman blank screen. It worked fine yesterday, I haven't actively allowed any updates though. running Ubuntu 18.04

Edit:
I resolved it by upgrading it to 7.23.
I used command line, and I suggest you do the same, to also get all the errors, if any.
sudo snap switch --channel=latest/stable postman
sudo snap refresh postman

And to run type,
postman

I had some postmans running from previous tries in the background which caused another different error after upgrading, but after killing them it finally worked.
